server/data/data-memory/src/test/java/org/apache/james/mailrepository/memory/MailRepositoryStoreImplTest.java

```diff
@@ -17,7 +17,7 @@
  * under the License.                                           *
  ****************************************************************/
```

Comment:
Can't this file be located in data-lib tests?

Reply:
Note that the current location is due to the use of MemoryUrlRepositoryStore.

I'm ok with the current location.